Error when typing any character

What happens:

This error bubble appears:

```
Error: Process exited with non-zero code: 1
Error: Process exited with non-zero code: 1
    at ChildProcess.<anonymous> (/home/user/.atom/packages/linter-pep8/node_modules/sb-exec/lib/index.js:52:22)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:850:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at Pipe._onclose (net.js:477:12)
```

What I do to make this happen:

1. Install atom for ubuntu 16.04.
2. `pip install pep8`. Requirement already satisfied (use --upgrade to upgrade): pep8 in ./.local/lib/python2.7/site-packages. 
3. Open a python file.
4. Type any character.

Versions:

Ubuntu 16.04
Atom 1.10.1
linter 1.11.16
linter-pep8 1.3.1
pep8 1.7.0

Btw, seeing errors like this in console:
```
[exec] Process exited with no-zero code, stdout:  stdin:19:2: E111 indentation is not a multiple of four
stdin:19:2: E113 unexpected indentation
```
The process executes but the wrapper seems to assume it exits with nonzero code, but it seems like both `pep8` and `pycodestyle` return nonzero code when there are errors, by design:
```
$ pep8 log_config.py
log_config.py:176:1: E302 expected 2 blank lines, found 1
$ echo $?
1
```

--- a/lib/main.coffee
+++ b/lib/main.coffee
@@ -35,7 +35,7 @@ module.exports =
           parameters.push("--ignore=#{ignoreCodes.join(',')}")
         parameters.push('-')
         msgtype = if atom.config.get('linter-pep8.convertAllErrorsToWarnings') then 'Warning' else 'Error'
-        return helpers.exec(atom.config.get('linter-pep8.pep8ExecutablePath'), parameters, {stdin: textEditor.getText()}).then (result) ->
+        return helpers.exec(atom.config.get('linter-pep8.pep8ExecutablePath'), parameters, {stdin: textEditor.getText(), ignoreExitCode: true}).then (result) ->
           toReturn = []
           regex = /stdin:(\d+):(\d+):(.*)/g
           while (match = regex.exec(result)) isnt null

